### DISCUSSION TITLE
RefinableDate## not showing on all results

### DISCUSSION THREAD

**Discussion by Gaztab** [microsoft-search/pnp-modern-search#2189] [Q&A]
Hi, 
Weird one and I want to know if I am missing something or if its possible it'll resolve itself. 

Background: 
I have a PnP results webpart which looks at a specific document library. I added a column "Date of issue" to 'RefinableDate02' (note: all items in the doc library have an entry in this column).
I gave it 24 hours for SharePoint to catch up with itself. I have now added "Refinable02" to the 'Date' field in the card layout. But only 1 item in the whole list is showing the 'Date of issue'. The rest are blank. 

Is it clear as to what I am missing here? 

Thanks in advance.

**Comment by wobba** [comment]
You have to re-index the library for search to pick up the cp/mp mapping. If not only new/changed items are picked up.

**Reply by Gaztab** [reply]
Great thank you. I've ran that now. Do you know roughly how long this process normally takes?

**Reply by wobba** [reply]
Anywhere from 5 minutes to 48h 🙂 Usually on the shorter side.

**Reply by Gaztab** [reply]
Awesome. Again, thank you!
